ports/esp32/modules/flashbdev.py

```diff
@@ -26,9 +27,8 @@ def ioctl(self, op, arg):
             return self.SEC_SIZE
 
 size = esp.flash_size()
-if size < 1024*1024:
+if size < 512*1024:
```

Comment:
Are you including the wear level size here? I'm not sure I see that.
https://github.com/micropython/micropython/pull/3576#issuecomment-412368432

Reply:
uhm... not really sure... but I guess this should be the minimum partition size, so this should be 528k... have to check this again, to be really sure